src/Components/Artist/MarketDataSummary/__tests__/__snapshots__/MarketDataSummary.test.js.snap

```diff
@@ -12,16 +12,18 @@ exports[`MarketDataSummary renders correctly 1`] = `
   className="c0"
 >
   <div>
-    $63m
+    $63,312,500
```

Comment:
Why is this changed?

Reply:
Not sure, was there some other change outside of my pr that could have affected this?  I didn't make any changes to the market data rendering.

Reply:
Actually this is because I am using a different fixture here: https://github.com/artsy/reaction/pull/699/files/13fb2ce08a6ac59da4db28d7bf33202eed683931#diff-9529e5842b7bcdd0f9b413d7120b6baeR13

Reply:
Are you returning the data rounded from the api?  this is what returned from metaphysics for me: https://github.com/artsy/reaction/pull/699/files/13fb2ce08a6ac59da4db28d7bf33202eed683931#diff-e9af1869999b3d835b6944bbbfe05a8cR928

Reply:
I think MP takes a parameter of `format` when `display` is requested there:

https://github.com/eessex/reaction/blob/13fb2ce08a6ac59da4db28d7bf33202eed683931/src/Components/Artist/MarketDataSummary/MarketDataSummary.tsx#L174

Not sure what changed in test data that we lost it here.

Reply:
ok in that case i am not using that parameter in the fixture, and will update to reflect that.